### ISSUE TITLE
SaltStack error on non-ISO install [20220403]

### ISSUE THREAD

**Issue by azcjtf** [Security-Onion-Solutions/securityonion#7694]
how do I resolve this error?

following:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

cloned latest repo, per instruction, started "so-setup-network"

fails due to salt stack

```
Ign:4 https://repo.saltproject.io/py3/ubuntu/20.04/amd64/archive/3004 focal InRelease
Err:6 https://repo.saltproject.io/py3/ubuntu/20.04/amd64/archive/3004 focal Release
  404  Not Found [IP: 99.86.38.46 443]
```
  
 *see attached sosetup.log file
[sosetup_x.log](https://github.com/Security-Onion-Solutions/securityonion/files/8405059/sosetup_x.log)

any help would be greatly appreciated

**Comment by 73F0SG1R** [comment]
i have this issue as well

**Comment by dougburks** [comment]
From https://blog.securityonion.net/2022/04/saltstack-security-release-causing.html:
> SaltStack Security Release Causing Security Onion Installations to Fail on Ubuntu
SaltStack has released version 3004.1:
https://docs.saltproject.io/en/latest/topics/releases/3004.1.html
>
> SaltStack also removed version 3004 which is causing new installations of Security Onion to fail on Ubuntu. For CentOS, we host our own packages so those installations are still working properly.
>
> We are working on a Security Onion hotfix to include SaltStack version 3004.1. We hope to release this hotfix next week.

In the meantime, you should be able to install using our Security Onion ISO image or perform a manual installation on CentOS 7.

If you have further questions or problems, please start a new discussion at https://securityonion.net/discuss.
